style.css

```diff
@@ -1,93 +1,98 @@
-html, body {
-	margin: 0px;
-	padding: 0px;
-	height: 100%;
-	font-family: sans-serif;
-	font-size: 10pt;
+header{
```

Comment:
about the header- it is designed be small and fit on mobile screens without requiring flex wrap. this layout won't fit on mobile screens and will look weird. try resizing the browser below 400px width.